src/components/resource/AdminMenu.jsx

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+import React from 'react/addons';
+import {DropdownButton, MenuItem} from 'react-bootstrap';
+import classNames from 'classnames';
+
+import 'font-awesome/css/font-awesome.css';
+import './AdminMenu.scss';
+
+
+const AdminMenu = React.createClass({
+
+  displayName: 'AdminMenu',
+
+  propTypes: {
+    className: React.PropTypes.string,
+    resourceText: React.PropTypes.shape({
+      text: React.PropTypes.string,
+      description: React.PropTypes.string,
+      editUrl: React.PropTypes.string,
+    }).isRequired,
+    isAdmin: React.PropTypes.bool,
+  },
+
+  render() {
+    var title = <i className='fa fa fa-puzzle-piece'></i>;
+    return (
+      <DropdownButton className={classNames(this.props.className, 'AdminMenu', 'transparent')}
+                      title={ title } noCaret="true">
```

Comment:
The `React.createElement` example is juste here to show you what the JSX compiles to. Boolean props work just like in HTML: if they are present but with no value, they are considered to be `true`.